hikari/impl/rest.py

```diff
@@ -676,8 +703,7 @@ async def _request(
                         start = time.monotonic()
 
                     # Make the request.
-                    session = self._acquire_client_session()
-                    response = await session.request(
+                    response = await live_attributes.client_session.request(
```

Comment:
What will happen if the live attributes are closed before this is done?

Reply:
Changed this so that _get_live_attributes is called multiple times in _request to account for it being closed mid-request